internal/otel/config/helpers/processors/batch_processor.go

```diff
@@ -17,6 +19,8 @@ var BatchProcessorID component.ID = component.NewID(batchProcessorName)
 func BatchProcessorCfg() *batchprocessor.Config {
 	factory := batchprocessor.NewFactory()
 	cfg := factory.CreateDefaultConfig().(*batchprocessor.Config)
+	cfg.SendBatchSize = 8192
```

Comment:
I do wonder if we'll hit more out of order here with this, but TBD

Reply:
It looks like this was actually the default anyways even when not set so I think we should be okay.

Reply:
Ah yeah I really meant both (but didn't line-select right). Like now the batches will have multiple samples (for the same active series / metric) all adjacent to one another in a single request. I assume that's fine (since AMP will re-order, and we order by timestamp)